webpack.config.js

```diff
@@ -1,43 +1,77 @@
+const webpack = require('webpack');
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const CleanWebpackPlugin = require('clean-webpack-plugin');
 
-const path = require("path");
-const HtmlWebpackPlugin = require("html-webpack-plugin");
+// Our directory path
+const BUILD_DIR = path.join(__dirname, 'dist');
+// App directory
+const APP_URL = path.join(__dirname, 'src');
+// Public directory
+const PUBLIC_URL = path.resolve(__dirname, 'public');
+
+// JS|JSX test regex
+const jsTest = /\.(js|jsx)$/;
+// CSS test regex
+const cssTest = /\.css$/;
+// Image test regex
+const imgTest = /\.(png|jpg|jpeg|gif|svg)$/i;
+// Files to clean
+const filesToClean = [
+  'dist',
+];
+
+// Our application vendor files
+const VENDOR_LIBS = [
+  'react',
+  'react-dom',
+  'react-router-dom',
+];
+
+/** Webpack Pluguins */
+
+// HTMLWebpackPlugin config
+const HWP = new HtmlWebpackPlugin({
+  template: `${APP_URL}/index.html`,
+  filename: `${BUILD_DIR}/index.html`,
+});
+
+/** nstance of the CleanWebpackPlugin */
+const CLEANPLUGIN = new CleanWebpackPlugin(filesToClean);
```

Comment:
Also, these plugins can be called directly without creating a constant to initialize it.
[You can check Webpack's documentation](https://webpack.js.org/plugins/html-webpack-plugin/#basic-usage)

Reply:
resolved